docs/content/docker.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "Docker Volume Plugin"
+description: "Docker Volume Plugin"
+---
+
+# Docker Volume Plugin
+
+## Introduction
+
+Docker 1.9 has added support for creating
+[named volumes](https://docs.docker.com/engine/reference/commandline/volume_create/).
+Since Docker 1.10 you can create named volumes through
+[Docker Compose](https://docs.docker.com/compose/compose-file/#volume-configuration-reference).
+As of Docker 1.12 volumes are supported by Docker Swarm, in particular in the
+[Compose v3 files](https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes)
+for *swarm stacks*. Note that the term *Swarm* here means the *docker swarm mode*
+included with Docker Engine, not the *standalone swarm* deprecated long ago.
+
+This is great for creating standalone volumes and easily connecting them to
+directories in containers as a way to share data between multiple containers.
+It also allows for the use of Docker Volume Plugins with stateful containers
+across multiple hosts. It can be useful to have persistent data on your host.
+
+Unlike the default `local` driver that comes with Docker,
+your data *will not be deleted* when the Volume is removed.
+With the `rclone` driver, if you remove the driver, and then recreate it
+later, any volume that was added to that volume will *still be there*.
+
+According to the [Docker Plugin API Docs](https://docs.docker.com/engine/extend/plugin_api/)
+plugins can be run managed by the docker daemon or as a native system service.
+The former mode is preferred, so we describe it first.
+
+## Running as Managed Plugin
+
+Docker daemon can install plugins from an image registry and run them managed.
+We maintain the
+[docker-volume-rclone](https://hub.docker.com/r/rclone/docker-volume-rclone/)
+plugin image on the [Docker hub](https://hub.docker.com).
+
+Before installing the plugin, let's create a couple required directories as
+plugin will *not* create them automatically:
+```
+mkdir -p /var/lib/docker/plugins/rclone/config
+mkdir -p /var/lib/docker/plugins/rclone/cache
+```
+
+Now install and enable the plugin:
+```
+docker plugin remove rclone
+docker plugin install rclone/docker-volume-rclone --grant-all-permissions --disable --alias rclone
+docker plugin set rclone RCLONE_VERBOSE=2 config=/etc/rclone args="--vfs-cache-mode=writes --allow-other"
```

Comment:
Plugin verbosity can be tweaked via `docker plugin set args="... -v..."` or `docker plugin set RCLONE_VERBOSE=...`. Arguments are more generic, they support tweaking not only verbosity but mount/vfs defaults too. Also note that only `RCLONE_VERBOSE` environment variable can be passed into plugin.

Reply:
One for the docs?